pyaedt/edb_core/edb_data/sources.py

```diff
@@ -316,6 +316,16 @@ def create_port_terminal(self, impedance=50):
         terminal.SetIsCircuitPort(True)
         return terminal
 
+    def delete(self):
+        """Delete active pingroup.
```

Comment:
```suggestion
        """Delete active pin group.
```
Given the use of the underscore (pin_group), I assume it should be pin group (two words).